bindings/wasm/crate/Cargo.toml

```diff
@@ -8,10 +8,11 @@ edition = "2018"
 crate-type = ["cdylib", "rlib"]
 
 [dependencies]
-wasm-bindgen = "0.2"
 js-sys = "0.3"
-xmtp-keystore = { path = "../../../crates/xmtp-keystore" }
+libxmtp-core = { path = "../../../crates/libxmtp-core" }
 once_cell = "1.17.1"
+wasm-bindgen = "0.2"
+xmtp-keystore = { path = "../../../crates/xmtp-keystore" }
```

Comment:
We can just nuke all the `xmtp-keystore` code in `bindings/wasm` at this point and start fresh. That can be done in a follow-up PR